compass/validate.py

```diff
@@ -176,8 +176,7 @@ def _compare_variables(variables, filename1, filename2, l1_norm, l2_norm,
         if 'Time' in da1.dims:
             time_range = range(0, da1.sizes['Time'])
             time_str = ', '.join(['{}'.format(j) for j in time_range])
-            print('{} '.format(variable)+(20-len(variable))*' '+
-                'Time index: '+time_str)
+            print('{} Time index: {}'.format(variable.ljust(20), time_str))
```

Comment:
@mark-petersen, here is a preferred way to do the string formatting you were doing.  I am pretty strongly opposed to the use of string "addition" in any context in python.